packages/web/src/store/application/ui/notifications/notificationsUISlice.ts

```diff
@@ -0,0 +1,48 @@
+import { Nullable } from '@audius/common'
```

Comment:
yeah the idea here is that instead of everything being "slice.ts", its the name of the slice, i see that's what's preferred in redux docs, and kinda something weve discussed with "saga.ts" being hard to work with as well. agreed it feels redundant given the directory naming though. im down to do whatever we think